### PR TITLE
feat: authorizenet checkout page styles

### PR DESCRIPTION
**Pull request by nehasacher143**
**Task:** Style the Authorizenet checkout page

**Description:** The styling for the checkout page is added.
Also, the HTML file is cleaned up

**Screenshot:**
screenshot-localhost_8001-2020 05 01-12_36_38

### CODE REVIEW

**Comment by neilLasrado** [comment]
@Alchez this needs to be sent upstream as well right?

**Comment by Alchez** [comment]
Yeah, I'll add it to the existing PR upstream

**Comment by Alchez** [comment]
Added to [existing upstream PR](https://github.com/frappe/erpnext/pull/21437).

File: erpnext/templates/pages/integrations/authorizenet_checkout.html [hunk 1]
```diff
@@ -3,49 +3,80 @@
...
+								placeholder="{{ _('****') }}" />
+						</div>
+
+					</div>
+
+				</div>
+				<div class="col-md-8 col-sm-12 submit_row">
+					<div class="row">
+					<button type="submit" class="submit" id="submit">{{_('Pay')}} {{ payment_context.amount }}</button>
+				</div>
```
Comment (Alchez) [inline erpnext/templates/pages/integrations/authorizenet_checkout.html]: Remove this extra `</div>`

File: erpnext/templates/pages/integrations/authorizenet_checkout.html [hunk 2]
```diff
@@ -3,49 +3,80 @@
 {% block title %} Payment {% endblock %}
 
 {% block script %}
-<script>{% include "templates/includes/integrations/authorizenet_checkout.js" %}</script>
 <script>
-	context= "{{ payment_context }}"
+	{
+		%
+		include "templates/includes/integrations/authorizenet_checkout.js" %
+	}
```
Comment (Alchez) [inline erpnext/templates/pages/integrations/authorizenet_checkout.html]: Put this in one line, it's throwing a SyntaxError while checking out:

```suggestion
	{% include "templates/includes/integrations/authorizenet_checkout.js" %}
```

**Review by Alchez — CHANGES_REQUESTED** [review]

File: erpnext/templates/pages/integrations/authorizenet_checkout.html (hunk 1 above)
Comment (nehasacher143) [reply erpnext/templates/pages/integrations/authorizenet_checkout.html]: There is no extra div, Its an indentation issue. Fixing it

**Review by neilLasrado — APPROVED** [review]

**Pull request merged**